starter-code/app.js

```diff
@@ -25,17 +25,21 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 // Authentication
 app.use(cookieParser());
-
+app.get("/", (request, response, next) => {
```

Comment:
You created a router file `routes/users.js`, but you didn't `require` it and `use` it in `app.js`. Instead, your routes are handled here as `app.get` etc. 
Make sure you remember to configure the routers on `app.js`.